[vtk] Disable Image formats we don't need

The list of supported (uncommented) image types seen in this patch
should be congruent with the *.cxx files listed in settings.bzl for
VTK::IOImage.

Reasoning for not upstreaming this patch: Drake-specific build option.

--- IO/Image/vtkImageReader2Factory.cxx
+++ IO/Image/vtkImageReader2Factory.cxx
@@ -179,23 +179,23 @@ void vtkImageReader2Factory::InitializeReaders()
 
   vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkPNGReader::New()));
   reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkPNMReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkPNMReader::New()));
+  // reader->Delete();
   vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkTIFFReader::New()));
   reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkBMPReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkBMPReader::New()));
+  // reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkSLCReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkSLCReader::New()));
+  // reader->Delete();
   vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkHDRReader::New()));
   reader->Delete();
   vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkJPEGReader::New()));
   reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkGESignaReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkGESignaReader::New()));
+  // reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkMetaImageReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkMetaImageReader::New()));
+  // reader->Delete();
-  vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkTGAReader::New()));
-  reader->Delete();
+  // vtkImageReader2Factory::AvailableReaders->AddItem((reader = vtkTGAReader::New()));
+  // reader->Delete();
 }
 
